frontend/packages/e2e/tests/page.test.ts

```diff
@@ -0,0 +1,15 @@
+import { expect, test } from '@playwright/test'
+
+const erdWebUrl =
+  process.env.ERD_WEB_URL ||
+  'http://localhost:3001/erd/p/github.com/mastodon/mastodon/blob/main/db/schema.rb'
+const cliUrl = process.env.CLI_URL || 'http://localhost:5173'
+
+const urls = [erdWebUrl, cliUrl]
+
+for (const url of urls) {
+  test(`Page has title for ${url}`, async ({ page }) => {
+    await page.goto(url)
+    await expect(page).toHaveTitle(/Liam ERD/)
+  })
+}
```

Comment:
It does not seem necessary to run it twice. GitHub Actions is called twice, so it is executed a total of 4 times.

Reply:
You were absolutely right.🙏  When running the ``pnpm test:turbo`` command manually (instead of through GitHub Actions), it would be great if the tests could run twice. So, I have adjusted the GitHub Actions workflow to ensure the tests run only once!
https://github.com/liam-hq/liam/pull/660/commits/f1b182dabee1a8efbb1a759caea195b02e381dd9

Reply:
@FunamaYukina 
I see, but Considering local debugging, a single run on the Playwright side is better. For example, if you want to debug only the web, you cannot control it. Other times, you may want to run tests in the preview environment from a local terminal.

- playwright side: Pass a URL and execute the test cases at that URL
- GitHub Actions side:
    - for Web: Run `pnpm test` passing the deployed web URL
    - for CLI: Run `pnpm test` passing the deployed cli URL
- (local debug): 
    - for Web debug: Run `URL=localhost:3000 pnpm --filter @liam-hq/e2e test`

This way, if we just execute the URL we receive playwright, it can be used for many purposes. Would you please reconsider?

Reply:
As you suggested, I've modified the script.🙏 https://github.com/liam-hq/liam/pull/660/commits/96728faccb3623af96c64ea2be0dd38eb0144379 

Instead of running ``pnpm test:turbo``, we can now execute specific tests using commands like ``URL=localhost:5173 pnpm test --filter @liam-hq/e2e`` for local development.
The command to run CLI tests is a bit lengthy😅: ``URL=localhost:3001/erd/p/github.com/mastodon/mastodon/blob/main/db/schema.rb pnpm test --filter @liam-hq/e2e``

Reply:
A bit lengthy, you're right😅 
Thanks for adjustment!